### PR TITLE
1071 add producible test

### PR DESCRIPTION
**Pull request by tjgersho**

### CODE REVIEW

**Comment by tjgersho** [comment]
https://github.com/SeedCompany/cord-api-v3/pull/1099
Done.

**Comment by tjgersho** [comment]
Can be merged after merging #1099

**Review by CarsonF — APPROVED** [review]
🚀

**Pull request merged**
